### PR TITLE
Map | LV:522 "Chances Claim" [MDB ignore]

### PR DESCRIPTION
**Pull request by spartanbobby**
This PR adds the map LV522 Chances claim as a votable option along with,
A new group of survivors exclusive to the map "USCM Force Reconnaissance" this set of survivors is meant to be harder than your normal survivors
A new CO WL'd role the FORECON Major whose objective is to keep his men alive (Rules regarding the FORECON major will be talked about with the CO council)
A small damage buff to the M1911 (two world wars)
a bunch of props for the map
A bunch of code changes that I honestly do not understand
It adds shale and some auto turfs
a bunch of audio files
And honestly there's been so much in this year-long project that I've probably forgotten something

## Why It's Good For The Game

I'd like to think that having a new map would make people enjoy the game more but I know you're just going to keep voting for LV and bigred anyway so

https://cdn.discordapp.com/attachments/458150341742166017/1018026590980018196/C-claim.png

## Changelog

:cl: SpartanBobby, Triodine, Geeves, Stan_Albatross, Legacy Spriters, Quickload,Morrow
add: added LV_522 Chances claim a new map
add: added USCM Force recon survivors and the special Force recon major for WL'd COs (exclusive to LV522)
add: gave special battle-damaged uniforms to members of force recon generously made by Quickload
add: added a bunch of wind and rain sounds for LV522 chances claim
add: added a bunch of props for LV522 chances claim
add: added some new autoturfs for LV522 chances claim
balance: buffed the damage of the M1911 Service pistol
/:cl:

### CODE REVIEW

**Comment by SuprisedBaby** [comment]
leonardo-dicaprio-cheers

**Comment by Chainsawicus** [comment]
+1 the perfect map to ignore and continue voting LV-Big Red.

**Comment by Joelampost** [comment]
one of the pillars of CMs new golden age

**Comment by stanalbatross** [comment]
-1 doesn't have five chungmillion ultrazine bottles and incin slugs

**Comment by johnravioliUSA** [comment]
👍

**Comment by Superreallycoolguy** [comment]
I will play survivor to just larp as the FORECON major, god bless you bobby

**Review by Agameofscones — APPROVED** [review]
Mapping checks out as do sprites.

Supporting map code for integration ok, get someone else to look at the recon guy stuff.

File: code/datums/skills.dm [hunk 1]
```diff
@@ -428,6 +428,109 @@ CIVILIAN
...
+		SKILL_CONSTRUCTION = SKILL_CONSTRUCTION_ENGI,
+		SKILL_MELEE = SKILL_MELEE_TRAINED,
+		SKILL_CQC = SKILL_CQC_TRAINED,
+		SKILL_FIREARMS = SKILL_FIREARMS_DEFAULT,
+		SKILL_MEDICAL = SKILL_MEDICAL_MEDIC,
+		SKILL_SURGERY = SKILL_SURGERY_NOVICE,
+		SKILL_ENDURANCE = SKILL_ENDURANCE_TRAINED,
+		SKILL_LEADERSHIP = SKILL_LEAD_NOVICE,
+		SKILL_VEHICLE = SKILL_VEHICLE_DEFAULT,
+			SKILL_JTAC = SKILL_JTAC_TRAINED
```
Comment (stanalbatross) [inline code/datums/skills.dm:464]:
```suggestion
		SKILL_JTAC = SKILL_JTAC_TRAINED
```

File: code/game/objects/items.dm [hunk 2]
```diff
@@ -181,9 +181,12 @@ cases. Override_icon_state should be a list.*/
 			if(MAP_WHISKEY_OUTPOST, MAP_DESERT_DAM, MAP_BIG_RED, MAP_KUTJEVO)
 				icon_state = new_icon_state ? new_icon_state : "d_" + icon_state
 				item_state = new_item_state ? new_item_state : "d_" + item_state
-			if(MAP_PRISON_STATION, MAP_PRISON_STATION_V3)
+			if(MAP_PRISON_STATION, MAP_PRISON_STATION_V3, MAP_LV522_CHANCES_CLAIM)
 				icon_state = new_icon_state ? new_icon_state : "c_" + icon_state
 				item_state = new_item_state ? new_item_state : "c_" + item_state
+//			if()
+//				icon_state = new_icon_state ? new_icon_state : "u_" + icon_state
+//				item_state = new_item_state ? new_item_state : "u_" + item_state
```
Comment (stanalbatross) [inline code/game/objects/items.dm:189]: erm what the scallop

File: code/game/objects/structures/props.dm [hunk 3]
```diff
@@ -18,6 +18,31 @@
...
 	bound_height = 96
+	var/on = FALSE//if this is set to on by default, the drill will start on, doi
+
+/obj/structure/prop/dam/drill/attackby(obj/item/W, mob/user)
+	. = ..()
+	if(isXeno(user))
+		return
+	else if (ishuman(user) && istype(W, /obj/item/tool/wrench))
+		on = !on
+		visible_message("Wrench the controls of [src]. The drill jumps to life." , "[user] wrenches the controls of [src]. The drill jumps to life.")
```
Comment (stanalbatross) [inline code/game/objects/structures/props.dm:29]:
```suggestion
		visible_message("You wrench the controls of \the [src]. The drill jumps to life." , "[user] wrenches the controls of \the [src]. The drill jumps to life.")
```
also this entire line makes no sense

File: code/game/objects/structures/props.dm [hunk 4]
```diff
@@ -18,6 +18,31 @@
...
+
+		Update()
+
+/obj/structure/prop/dam/drill/proc/Update()
```
Comment (stanalbatross) [inline code/game/objects/structures/props.dm:33]: do not use capital letters in proc names

File: code/game/objects/structures/props.dm [hunk 5]
```diff
@@ -738,6 +763,36 @@
 	name = "M1 pattern festive needle torus"
 	desc = "In 2140 after a two different sub levels of the São Luís Bay Underground Habitat burned out (evidence points to a Bladerunner incident, but local police denies such claims) due to actual wreaths made with REAL needles, these have been issued ever since. They're made of ''''''pine'''''' scented poly-kevlon. According to the grunts from the American Corridor, during the SACO riots, protestors would pack these things into pillow cases, forming rudimentary body armor against soft point ballistics."
 	icon_state = "wreath"
+/obj/structure/prop/vehicles/
```
Comment (stanalbatross) [inline code/game/objects/structures/props.dm:766]:
```suggestion
/obj/structure/prop/vehicles
```

File: code/modules/gear_presets/forcon_survivors.dm [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+
+/datum/equipment_preset/survivor/forecon/proc/add_forecon_weapon_pistol(mob/living/carbon/human/H)
+	var/random_pistol = rand(1,6)
+	switch(random_pistol)
+		if(1 , 2 , 3 , 4)
+			H.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3(H), WEAR_WAIST)
+			H.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/m1911(H), WEAR_IN_BELT)
+			H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/m1911(H), WEAR_IN_BELT)
+			H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/m1911(H), WEAR_IN_BELT)
+		if(5 , 6)
```
Comment (stanalbatross) [inline code/modules/gear_presets/forcon_survivors.dm:69]:
```suggestion
	var/random_pistol = rand(1,3)
	switch(random_pistol)
		if(1 , 2)
			H.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3(H), WEAR_WAIST)
			H.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/m1911(H), WEAR_IN_BELT)
			H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/m1911(H), WEAR_IN_BELT)
			H.equip_to_slot_or_del(new /obj/item/ammo_magazine/pistol/m1911(H), WEAR_IN_BELT)
		if(3)
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Comment by hotimep** [comment]
CO survivor will be real in 5 seconds

**Comment by stanalbatross** [comment]
no map approval until it's playtested

File: code/game/gamemodes/colonialmarines/colonialmarines.dm [hunk 7]
```diff
@@ -186,18 +186,21 @@
...
+
+						var/input = "Security lockdown will be lifting in 30 seconds per automated lockdown protocol."
+						var/name = "Automated Security Authority Announcement"
+						marine_announcement(input, name, 'sound/AI/commandreport.ogg')
+						for(var/i in GLOB.living_xeno_list)
+							var/mob/M = i
+							sound_to(M, sound(get_sfx("queen"), wait = 0, volume = 50))
+							to_chat(M, SPAN_XENOANNOUNCE("The Queen Mother reaches into your mind from worlds away."))
+							to_chat(M, SPAN_XENOANNOUNCE("To my children and their Queen. I sense the large doors that trap us will open in 30 seconds."))
+						addtimer(CALLBACK(src, .proc/open_podlocks, "map_lockdown"), 300)
```
[the remark this answers is not in the document]
Comment (spartanbobby) [reply code/game/gamemodes/colonialmarines/colonialmarines.dm:203]: Makes it a bit more readable

**Comment by c4xmaniac** [comment]
You should still add the nightmare 1% spawn chance role of a chef with the oversized pizza cutter :3

**Comment by stanalbatross** [comment]
fix LZ2 being weedable

fix LZ2 being unusable

**Comment by stanalbatross** [comment]
image

boring area

**Comment by JediKing112** [comment]
absolutely terrible map, pointless entities and huge amounts of lag. Many areas and hallways are blocked off for no reason, the windows let marines CAS, the podlocks open too late and at 35 for an unknown reason. The eastern reactor is a dead-end made to trap xenos for no reason as podlocks come down, and the U-Shaped clearly typical hive location is AWFUL to navigate and defend. This map needs a lot of work.

**Comment by S1tr3p** [comment]
holy shit who made this map this map is fuckin AWESOME!!!!

**Comment by SuprisedBaby** [comment]
> absolutely terrible map, pointless entities and huge amounts of lag. Many areas and hallways are blocked off for no reason, the windows let marines CAS, the podlocks open too late and at 35 for an unknown reason. The eastern reactor is a dead-end made to trap xenos for no reason as podlocks come down, and the U-Shaped clearly typical hive location is AWFUL to navigate and defend. This map needs a lot of work.

You played this map during 200 pop CM when even 160 pop is extremely laggy.

**Comment by Smov1** [comment]
Make the map more APC friendly. My driver literally killed himself 10 minutes in, since there was barely any open passages for the APC

**Comment by stanalbatross** [comment]
lz2 weedable

**Comment by SexmasterWasTaken** [comment]
Add more unique loot, makes it fun. Good map. Make the shutters open way earlier, though. Maybe at like 12:25?

**Comment by Necrosium** [comment]
Besides the issue with the lag and the general confusion of the map (just because i didn't know how it was shaped, just me being stupid), I'm curious whether not being able to weed the grey concrete areas with hardy weeds is intended or not. Perhaps that's something to fix too.

**Comment by S1tr3p** [comment]
> > absolutely terrible map, pointless entities and huge amounts of lag. Many areas and hallways are blocked off for no reason, the windows let marines CAS, the podlocks open too late and at 35 for an unknown reason. The eastern reactor is a dead-end made to trap xenos for no reason as podlocks come down, and the U-Shaped clearly typical hive location is AWFUL to navigate and defend. This map needs a lot of work.
> 
> You played this map during 200 pop CM when even 160 pop is extremely laggy.

i played this map because its FUCKING AWESOMER!!!!

**Comment by SilverBerryWave** [comment]
I have to question why there's a sensor tower is placed on the marine's side of the map, this just ensures that before the podlocks open marines will be able to fix it and rush the hive. I'd consider either removing it or putting it somewhere else the marines have to work to get too.

Other than that, the map is quite large. Performance was pretty bad but it had a large player count. The unbreakable windows were kinda silly just because you had xenos and marines staring at each other. Puddles were a weird, over saturated blue. 

But I actually like the map overall and I hope it makes it into rotation, could just use some touch ups and balancing.

**Comment by Shad0vvs** [comment]
> EDIT: Also, apparently xenos couldn't drag prebursted bodies. I dunno what's up with that.

Some admin hit a nono button during the round.

**Comment by stanalbatross** [comment]
replace one of the ar10s with an m16 ar10s are meant to be rare

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 8]
```diff
@@ -62,3 +62,30 @@
 	roundstart_damage_times = 2
 
 
+//Military Survivors//
+
+/obj/effect/landmark/survivor_spawner/lv_522/lv522_forecon_major
+	equipment = /datum/equipment_preset/survivor/forecon/major
+	intro_text = list("<h2>You are the commanding officer of the USS Hanyut</h2>",\
```
Comment (XSlayer300) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:69]:
```suggestion
	intro_text = list("<h2>You are the commanding officer of the USS Hanyut.</h2>",\
```

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 9]
```diff
@@ -62,3 +62,30 @@
...
+	"<span class='notice'>You are VERY aware of the xenomorph threat</span>")
```
Comment (XSlayer300) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:70]:
```suggestion
	"<span class='notice'>You are VERY aware of the Xenomorph threat.</span>")
```

File: code/game/objects/effects/landmarks/survivor_spawner.dm [hunk 10]
```diff
@@ -62,3 +62,30 @@
...
+	story_text = "This entire operation went shits up when your team investigated the atmospheric processor two weeks ago, from that point on you've been surviving by the skin of your teeth waiting your ships automated distress beacon to be picked up...until now"
```
Comment (XSlayer300) [inline code/game/objects/effects/landmarks/survivor_spawner.dm:71]:
```suggestion
	story_text = "This entire operation went shits up when your team investigated the atmospheric processor two weeks ago, from that point on you've been surviving by the skin of your teeth waiting your ship's automated distress beacon to be picked up... until now."
```

File: maps/LV522_Chances_Claim.json [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+        "/datum/equipment_preset/survivor/forecon/standard"
+    ],
+    "defcon_triggers": [
+        3300,
+        2100,
+        1450,
+        580,
+        0.0
+    ],
+    "survivor_message": "You are USCM Force Reconnaissance (FORECON) marine, stranded on this planet after your team was wiped out while investigating the distress signal. With your pilot dead along with the majority of your unit you and your team have been running and gunning to survive..until now",
```
Comment (XSlayer300) [inline maps/LV522_Chances_Claim.json:15]:
```suggestion
    "survivor_message": "You are USCM Force Reconnaissance (FORECON) marine, stranded on this planet after your team was wiped out while investigating the distress signal. With your pilot dead along with the majority of your unit, you and your team have been running and gunning to survive... until now.",
```

**Comment by cnaper-dodo** [comment]
I tried the map as for survivors.Creator.I have a question for you.How do you expect to survive here.Yes, a good starting weapon and equipment.That's all.I didn't quite understand about the roles.The idea about the intelligence corps is interesting.But there is NO way to be an engineer with 2 medical supplies and no tools.
A short dash across the map showed a couple of interesting things:
IT is HUGE
A couple of broken ships (it would be interesting if the survivors could fix them and fly away)
A broken APC (there are no wheels anywhere on the map)
An interesting survivor's bunker (it is also almost impossible to get there)
Lack of means for the survival of survivors.There are no materials, cartridges, medicines and special tools (the PMC mini kit does not count)
Florina survives at the expense of turrets
Kutevo survives at the expense of tels and good weapons
LV-624 survives due to competent positional play (and good saturation)
Sorokina survives due to the HUGE supply of loot
Solars survive at the expense of positioners (And now miners)
----
There is also an interesting 3.5 object.And a good initial bow.All.The whole map is riddled with bullets.I didn't find the objects needed for survival anywhere outside.Yes, there is something in the boxes.....But we have only 2 minutes to 8 RUNNERS.8 CARL.and we have practically nothing to resist them.
My personal opinion:The map is interesting, but requires improvement for the survivors.They have too little time to find the initial things for defense.t1 defense.How they survive further, I don't know yet.My opinion is that the saturation of loot is needed, as in Sorokin.
Also, a sophisticated aggressive squad of otters (CLF/UPP) is appropriate here.They will also have something to do.Complete the map with transport, turrets, weapons and ammunition, and we will have an interesting map.

**Comment by Mister-moon1** [comment]
the map falls into the same issue of corsat, too laggy, too big and too confusing to get anything done, literally spent 9 minutes to walk around in a circle and didnt see a single xeno, theres not a clear direction of where to go

**Comment by Zenith00000** [comment]
Good so far, nice for LZ holds. Just need more testing to really get a better idea of it.

**Comment by Hamyrappy** [comment]
This map is too complex IMHO. Mazes, mazes, mazes. But it is kinda funny

**Comment by stanalbatross** [comment]
> the map falls into the same issue of corsat, too laggy, too big and too confusing to get anything done, literally spent 9 minutes to walk around in a circle and didnt see a single xeno, theres not a clear direction of where to go

- go north
- the server was at 200 players, it was not the map
- it's only 15 tiles longer than LV 
- look at a picture of the map

**Comment by sose96** [comment]
Is there a way to remove the ledges around the LZs? You can’t build cades near them and it makes defending a real pain in the ass.

**Comment by MusselMain** [comment]
> ## I tried the map as for survivors.Creator.I have a question for you.How do you expect to survive here.Yes, a good starting weapon and equipment.That's all.I didn't quite understand about the roles.The idea about the intelligence corps is interesting.But there is NO way to be an engineer with 2 medical supplies and no tools.
> A short dash across the map showed a couple of interesting things:
> IT is HUGE
> A couple of broken ships (it would be interesting if the survivors could fix them and fly away)
> A broken APC (there are no wheels anywhere on the map)
> An interesting survivor's bunker (it is also almost impossible to get there)
> Lack of means for the survival of survivors.There are no materials, cartridges, medicines and special tools (the PMC mini kit does not count)
> Florina survives at the expense of turrets
> Kutevo survives at the expense of tels and good weapons
> LV-624 survives due to competent positional play (and good saturation)
> Sorokina survives due to the HUGE supply of loot
> Solars survive at the expense of positioners (And now miners)
> There is also an interesting 3.5 object.And a good initial bow.All.The whole map is riddled with bullets.I didn't find the objects needed for survival anywhere outside.Yes, there is something in the boxes.....But we have only 2 minutes to 8 RUNNERS.8 CARL.and we have practically nothing to resist them. My personal opinion:The map is interesting, but requires improvement for the survivors.They have too little time to find the initial things for defense.t1 defense.How they survive further, I don't know yet.My opinion is that the saturation of loot is needed, as in Sorokin. Also, a sophisticated aggressive squad of otters (CLF/UPP) is appropriate here.They will also have something to do.Complete the map with transport, turrets, weapons and ammunition, and we will have an interesting map.

The issue of runners comes from both the incredibly above average number of players those rounds (200 instead of the average 90-110), and honestly, it's nice to see a map that forces the survivors to play a roam heavy playstyle for once. In the case of the nonfunctional ships, don't hold your breath, there's been nonfunctional ships on half the playable maps. Survivors are meant to be a challenging role, and since Xeno mains started realizing that they were fully capable of attacking the survivors with the whole hive, it's become even more challenging. HOWEVER, this is the appeal and the only reason why players are having trouble with this new map is due to them not knowing where loot is/ where the best holding positions are, just like when Kutjevo was new and I had to learn the map (for a few rounds all the survivors were getting killed pretty badly as they didn't know the map yet). My final point is, please for the love of honkmother learn to spell, reading the bold text was pretty painful (IF ENGLISH IS YOUR SECOND LANGUAGE IGNORE MY PREVIOUS STATEMENT).

**Review by stanalbatross — CHANGES_REQUESTED** [review]
- apcs should start with much less charge
- fix underwear

**Comment by Yellows91** [comment]
It's a good map but might be too open for xeno combat to work.

Unless xenos have alot of drones and hivelords who are willing to build its gonna be a struggle

**Comment by c4xmaniac** [comment]
Needs weapon/ammo spawns on the map, or alternatively survivors to start with more ammo for their weapons. Run out long before you should rn. There's very few weapons atm, the natural rally spot for weapons/ammo is sec, but sec has 2 mars with no extra mags.

**Comment by PalJohnson** [comment]
Survivor Gameplay Feedback

I wanted to say that what you are working on here is an exciting idea for a map and I encourage you to keep working on it so that it may reach its fullest potential. I played a few rounds as a survivor on the map and had some feedback regarding survivor gameplay. In short, the map needs better loot.

From what I understand the survivors here are a part of an elite recon group that found themselves in a bad situation when they came into contact with the xenomorphs. Everyone spawning together is an excellent idea perhaps you might want to make more nightmare inserts across your map with a different rally spot. The spawn allowed us to coordinate a bit better especially since a lot of the people playing at the time were new to the map and some people were inexperienced in the role of survivor. For the majority of the map, we found ourselves running around and attempting to find any gear that may aid us in our endeavor to wait it out until the marines eventually arrived. The only real things that we were able to scavenge were parkas, helmets, and medical supplies which were located south of the spawn point. It goes without say based off the gear we spawned with, having looked at the code, and the map itself you intend to survivors to really be scared for their life.

I like this. I won't lie when I say that survivors have a little to much gear and Ive worked overtime to reduce it and randomize it as frequently as I can in order to keep things fresh and exciting while at the same time not leaving players out to dry. Survivors ideal should be dying 75% of the time given that most of the time they are civilians in an a very precarious situation. The lights go out, their co-workers are missing, and the place goes silent as terror can be felt for their impending doom. Your map works perfectly in this aspect setting the scene for the environment however I think there is an issue of balance that needs to be addressed to make things fun.

It goes without saying you want survivors to roam around the map for gear yet there is little to no gear. You have said yourself that these are elite individuals who should rely on the wit and the gear they have to survive which I totally get but I don't your storytelling has come at a cost for gameplay and ultimately fun for people who play survivor. The role itself is not flawed but the environment in which it was fostered has pushed it in a direction that I don't think your map takes into consideration.

Foraging is key to survival on this map and it is a requirement for any survivor to live regardless of the map but if there is very little to no gear to scavenge then you are better off hiding in a locker. On the map if I recall there are 3 bolt action hunting rifles, 2 mar 40s with 4 magazines, 1 M41AE LMG with 1 spare magazine, 1 m2c and the remnants of a smart gun kit with 2 spare drums with various degrees of bullets, Flame thrower,and a combat shotgun. Half of the viable weapons located on the map are currently behind inaccessible barriers due to the power being out on the majority of the map. 

Even if you wanted to try to fix the apcs there like outdoor storage to access items such as C4s to blow up walls to get to potentially better loot you are dependent upon an engineer which is an rng spawn. Survivors on the map don't even spawn with tool pouches let alone basic engineering skills that would enable them to complete the task set forth by yourself to reward them for roaming and finding good gear like the ar-10 or survivor mask. Time is everything for survivors and every second they spend having to run across the map (assuming they know where the loot spawns are) and jumping through the various hurdles to get to a piece of gear that may or may not help them is just bad gameplay. Xenos begins crawling out of tunnels (the majority of which span the center of the map) by 12:06 having evolved to T1s and the current map doesn't allow survivors a fighting chance to defend themselves adequately.

My only critique of the gear presets for the survivors is the lack of available gear to loot to facilitate. Each survivor spawns with either m41a, l42a, and m39 which are ok weapons however the lack of ammo to replenish their guns leaves them at a huge disadvantage. With defenders able to sponge attacks ammo goes by quickly especially when you consider queen heals. From my rounds playing most survivors are having to ditch their primary weapons by min 10 due to having spent them all leaving them with only lackluster colony pistols (with the exception of m1911) that haven't been updated to be remotely viable. If you want survivors to be able to survive and live to tell the tale of the elite recon force then give them a few more rounds to fire if you don't plan on adding more guns. 

Holdouts are non-existent. The dependency of the engineer spawn forces survivors to be on the move which I don't think is necessarily a good thing. The map doesn't really feature a place where survivors could fight in and have a bit of a last stand due to the lack of pre-built fortifications. The lack of engineering skills among the survivors as stated previously means loot like the ar-10, c4s (which I assume is there to open up other loot) and others is inaccessible. Survivors due to the time constraints, lack of engineering skills, lack of available tools, and materials forced survivors down a very linear way of gameplay.

In general, I think survivors for this map in its current state will lead to an unsatisfactory gameplay loop which might discourage people from playing due to how little there is in the player agency to enable them to survive. There needs to be a little more ammo and lootable items across the map in order to encourage survivors to explore the map to try to scavage what gear is available. If you don't plan to add more ammo or weapons to the map then at the very least give survivors more ammo and better kits so that they can actually have a slim chance to survive. There needs to be a balance between fluff and gameplay in order to have a satisfactory run as a survivor on this map. Your map has a story to tell through the lives of the recon commandos and if they can't get a fighting chance no matter how slim it is they won't be able to tell it.

**Comment by stanalbatross** [comment]
Stale exempt due to breaks between rounds of testing

**Comment by spartanbobby** [comment]
I've made various changes to the map including the following
-Overall the map should now be smaller but it does still use the majority of its space the map is roughly 20 tiles longer than LV
-Weapons and ammo have been added to the map for survivors to use but getting to it may require you to complete objectives such as finding C4 to blow open padlocks etc
-More overall detailing around the map to clear dead space
-LZ1 should be easier to hold

I'm open to T-merging this map once again when it suits the dev-team best

**Comment by Strobia** [comment]
my friend was decapped on the map and said '' It's just like the movies '' in dchat +1

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Unable to deploy or build in the downed dropships.
APC shouldn't be able to be moved at all, including rotating as it prevents xeno from getting out after getting in.
Reminder about the nondestroyable podlocks in the west.
Puddle sprite is still needed looks like.

**Comment by FoxyStalin** [comment]
After a survivor round, I enjoy it! Very cinematic experience.

Notes:
-Very ledge heavy, this is my biggest issue.  Add a bunch of staircases or remove a bunch of ledges,  particularly around the LZs.
- Allow building on the crashed dropships.  No reason not to be able to do that, especially if you intend survivors to try and hold at the northwest one.
- Give Synthsurvs boots that you can draw a knife from, please.  
- West LZ2 is very, very open.  It's like the issue Trijent LZ1 South has but even bigger, honestly.  I'd suggest adding in a few stone walls or something to tighten it up.   It'll be miserable in Lowpop otherwise,  which I suppose isn't the WORST thing, but, we'll have to see if LZ1 has simmilar problems or if that'd be a viable lowpop LZ.

**Comment by Sernoh** [comment]
Some feedback as Xeno, very early impressions so take it with several grains of salt:

- LZ2 north side should be broken up with walls/rocks on the marine side close to LZ, it's very wide open making it difficult to assault as Xenos, even with two boilers.
- Colony dusty tiles is a good anti-weed mechanic for the marine side of the map. However as Queen it's much harder to spread weeds on ovi. LV has roads, BR has side paths alongside buildings. Please consider adding one tile weedable lines through the dusty rocks purely so it's easier to weed from ovi.

Interesting map and I'd love to see it more in rotation

**Comment by Mollwu** [comment]
More than 2/3 of the map CANNOT be acessed by the apc and t heres a bug with the orange tiles not letting the apc pass trough. The parts of the map who we have acess have really major problems.

I linked most of the problems with the image, but really, it has some really really bad spots a literal "Death Road" for both marines and apc, and almost 55%+ of  the map is unacessible. C4 dont help ANYTHING because of the railings.
lv 522 apc

**Comment by Mollwu** [comment]
> More than 2/3 of the map CANNOT be acessed by the apc and t heres a bug with the orange tiles not letting the apc pass trough. The parts of the map who we have acess have really major problems.
> 
> I linked most of the problems with the image, but really, it has some really really bad spots a literal "Death Road" for both marines and apc, and almost 55%+ of the map is unacessible. C4 dont help ANYTHING because of the railings. lv 522 apc

I created a github account just to post this thing that was really pestering me. The stone wall of the death road is unbreakable by all means also.

**Comment by Nanu308** [comment]
Runtimes on your map/branch.

I can't replicate it on Master so i believe it is an issue stemming from this PR, it needs to be fixed prior to another TM.

bilde

**Comment by spartanbobby** [comment]
I've gotten alot of good data from the last 3 tests, I'd like to request a stop to the tests for the moment while I work away on the map a little more, I should have something testable within (Timeframe)

**Comment by johnravioliUSA** [comment]
This shit look like a spartanbobby map

**Comment by spartanbobby** [comment]
Chances Claim is once again ready for play-testing above 130 pop

**Comment by morrowwolf** [comment]
Synth survivors being spawned in as USCM marines with synth name/description.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Still going to run the test but the flipped tables aren't functioning correctly. They don't seem to have the correct flags even though visually flipped.

**Comment by ThePiachu** [comment]
Could we get a few `/obj/structure/machinery/recharge_station`s around on this map please? They are very handy for Synths and Synthvivors. Engi, Garage, Mining, Medical, and somewhere in the Reactor might be nice spots for them.

**Comment by ThePiachu** [comment]
The Secure Evidence Lockers (A-Block Security area) has access ACCESS_MARINE_BRIG, should also include ACCESS_CIVILIAN_BRIG so survivors like the Synth can access it.

**Review by morrowwolf — APPROVED** [review]
Approving both map and code.

**Pull request merged**